### ISSUE TITLE
Slash randomly added between url domain and path when triggering webhook

### ISSUE THREAD

**Issue by startork** [strapi/strapi#20423]
### Required System information
- Node.js version: v20.10.0
- NPM version: v10.2.5
- Strapi version: 4.19.0
- Database: mysql
- Operating system: linux-x64
- Is your project Javascript or Typescript: Typescript

### Describe the bug

Triggered webhooks sometimes post to the wrong URL. An extra slash is added to the beginning of the url path. So if the webhook url is defined as https://prod-application.co.uk/documents/publish Strapi will (seemingly randomly) try and POST to https://prod-application.co.uk//documents/publish instead.

### Steps to reproduce the behavior

1. Go to 'Settings > Webhooks' in the Strapi admin panel
2. Create a new webhook and enable 'update' and 'publish'
3. Trigger updates or publish new content and monitor the requests coming into your application api
4. Some will randomly be sent to ...co.uk//path-to-api instead of ...co.uk/path-to-api

### Expected behavior

Webhooks must be sent to the exact url set every time.

### Screenshots

Screenshot from our application papertrail logs
![image](https://github.com/strapi/strapi/assets/20688660/61a6914b-76ae-42a4-859d-41f10f0970dd)

**Comment by olitocher** [comment]
Hi, I am also struggling with this issue. We cannot get to the bottom of it but it is quite heavily impacting our clients so is relatively pressing

**Comment by startork** [comment]
Hi @derrickmehaffy 

Small update. After investigating further and discussing with client's. It could be that the double slash came from a previous typo in the webhook which has since updated. If this is the case is there anywhere this value could be stored outside of the database for Strapi to use? The strapi application has been restarted multiple times since webhooks were last updated so this shouldn't be the root of the issue. I have also checked the strapi_webhooks table in the database and everything looks correct there.

Thanks,
Starsky

**Comment by startork** [comment]
Turns out the issue was related to the above. We are using pm2 to run two instances of Strapi ensure zero downtime etc. We updated the webhook assuming it would update the value for both instances but this was not the case. We simply reloaded all instances to fix the issue.

The only thing to change here may be to check if the DB/ in-store values match periodically but as this isn't a bug I will close the issue.

Thanks
